### DISCUSSION TITLE
NO PERMITE ACCEDER A LAS OPCIONES

### DISCUSSION THREAD

**Discussion by DarioNagore** [chamilo/chamilo-lms#5876] [General]
**Describe**
Luego de la Instalación, desde el menu principal no permite acceder a las distintas opciones.

**To Reproduce**
 Las opciones llevan al INDEX.PHP inicial mostrando el final script "Este es link", sin ninguna otra opción.
**Expected behavior**
Se espera poder a las opciones para crear Categoria de Curso y Crear Curso

Chamilo 1.11.26
PHP 7.4

![image errores](https://github.com/chamilo/chamilo-lms/assets/175167731/2649aa7d-5810-45c5-a996-4d71cb6f69fd)

**Comment by ywarnier** [comment]
Hola @DarioNagore 
Parece que usas una sub-carpeta para tu portal Chamilo. No lo recomendamos porque dificulta la configuración del servidor web para esta ruta, pero en todo caso y para encontrar el problema, deberías asegurarte del valor de los parametros `root_web` y `url_append` dentro de tu `app/config/configuration.php`. Sospecho que están mal.
En tu caso, deberían ser así:
```
/**
 * Directory settings.
 */
// URL to the root of your Chamilo installation, e.g.: http://www.mychamilo.com/
$_configuration['root_web'] = 'https://www.idnagore.com.ar/';
// Path to the webroot of system, example: /var/www/
$_configuration['root_sys'] = '{algo interno a tu servidor, probablemente no sea necesario modificarlo}';
// Path from your WWW-root to the root of your Chamilo installation,
// example: chamilo (this means chamilo is installed in /var/www/chamilo/
$_configuration['url_append'] = 'aulaitec';
```

Si te funciona, por favor regresa aquí para decirlo (para que podamos cerrar este ticket).

**Comment by DarioNagore** [comment]
No funciona...lamentablemente. Se accede con el usuario, aparece la lista de opciones de Administración.. pero cada opción que intenta ser seleccionado, lleva a la cola del script **index.php,** mostrando "Este es el acceso al Link Este es el acceso al Link"
